libcontainer/cgroups/fscommon/rdma.go

```diff
@@ -0,0 +1,121 @@
+// +build linux
+
+package fscommon
+
+import (
+	"bufio"
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/configs"
+	"golang.org/x/sys/unix"
+)
+
+// parseRdmaKV: parses raw string to RdmaEntry.
+func parseRdmaKV(raw string, entry *cgroups.RdmaEntry) error {
+	var value uint64
+	var err error
+
+	parts := strings.SplitN(raw, "=", 3)
+
+	if len(parts) == 2 {
+		if parts[1] == "max" {
+			value = math.MaxUint64
+		} else {
+			value, err = strconv.ParseUint(parts[1], 10, 32)
+			if err != nil {
+				return err
+			}
+		}
+		if parts[0] == "hca_handle" {
+			entry.HcaHandles = uint32(value)
+		} else if parts[0] == "hca_object" {
+			entry.HcaObjects = uint32(value)
+		}
+	} else {
```

Comment:
maybe use short-circuiting here?

```
if len(parts) != 2 {
    return errors.New("Unable to parse RDMA entry")
}

k, v := parts[0], parts[1]
...
}
```

Reply:
@fuweid Resolved in latest commit.

Reply:
Do you push the commit for this?

Reply:
@fuweid oh i forgot `git add` on the file. Just added now.